src/code/dialogs/mergeDialog.js

```diff
@@ -126,6 +158,83 @@ const MergeDialog = WDialog.extend({
 
     return rebaseMessage;
   },
+
+  formatChanges: function (changes, origin, operation) {
+    const sortable = new Sortable();
+    sortable.fields = [
+      {
+        name: " ",
+        value: (e) => e.type,
+        format: (cell, value) => {
+          cell.textContent = value;
+        },
+      },
+      {
+        name: " ",
+        value: (e) => {
+          let v = "";
+          if (e.data.type === "link") {
+            v = e.data.link.ID;
+          } else if (e.data.type === "portal") {
+            v = e.data.portal.id;
```

Comment:
OK, I will make a sweep and make portal.id == portal.ID as well. All these stupid inconsistencies that have been there since 0.01 are maddening

Reply:
Distinction makes almost sense since `.ID` is wasabee specific while `.id` is from intel.